package.json

```diff
@@ -27,7 +27,8 @@
   ],
   "dependencies": {
     "request": "latest",
-    "debug": "~0.7.2"
+    "debug": "~0.7.2",
+    "mustache": "^2.2.1"
```

Comment:
When you add a dependency always add it to the package.json (use `npm i -S mustache` or for devDependencies `npm i -D mocha`)